app/src/main/res/layout/app_version_activity.xml

```diff
@@ -29,13 +29,24 @@
       app:titleTextColor="@color/white" />
   </com.google.android.material.appbar.AppBarLayout>
 
+  <View
+    android:id="@+id/app_version_shadow_view"
+    android:layout_width="0dp"
+    android:layout_height="4dp"
+    android:layout_gravity="top"
+    android:background="@drawable/toolbar_drop_shadow"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/app_version_app_bar_layout"
+    app:layout_constraintBottom_toTopOf="@id/app_version_fragment_placeholder"/>
+
   <FrameLayout
     android:id="@+id/app_version_fragment_placeholder"
     android:layout_width="match_parent"
     android:layout_height="0dp"
     app:layout_constraintBottom_toBottomOf="parent"
     app:layout_constraintEnd_toEndOf="parent"
     app:layout_constraintStart_toStartOf="parent"
-    app:layout_constraintTop_toBottomOf="@id/app_version_app_bar_layout"
+    app:layout_constraintTop_toBottomOf="@id/app_version_shadow_view"
```

Comment:
This constraint should not change as it will mean that the fragment will start below the shadow, but instead it should start below toolbar and the shadow should be on top of it.

Reply:
Done